scripts/dashboard/WalletBalance.vue

```diff
@@ -130,11 +130,13 @@ const emit = defineEmits([
 
 getEventEmitter().on(
     'transparent-sync-status-update',
-    (i, totalPages, finished) => {
-        const str = tr(translation.syncStatusHistoryProgress, [
-            { current: totalPages - i + 1 },
-            { total: totalPages },
-        ]);
+    (i, totalPages, finished, message) => {
```

Comment:
```suggestion
    (i, totalPages, finished, error) => {
```
Can you call this error/warning?